tools/nni_cmd/nnictl_utils.py

```diff
@@ -259,6 +274,15 @@ def trial_ls(args):
         response = rest_get(trial_jobs_url(rest_port), REST_TIME_OUT)
         if response and check_response(response):
             content = json.loads(response.text)
+            if args.head:
+                assert int(args.head) > 0, 'The number of requested data must be greater than 0.'
+                args.head = min(int(args.head), len(content))
```

Comment:
why use `int(args.head)`?

Reply:
Sorry, I don't know if specify the arg's type as int, it will be converted to int.